.idea/runConfigurations/android.xml

```diff
@@ -1,33 +0,0 @@
-<component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="android" type="AndroidRunConfigurationType" factoryName="Android App" activateToolWindowBeforeRun="false">
```

Comment:
these deletions shouldn't be pushed i think. Same goes for all runconfigurations